.github/workflows/ruby.yml

```
@@ -1,89 +1,32 @@
----
-name: Ruby Testing
 
-# Controls when the action will run.
-on:
-  pull_request:
+---
+name: Ruby Tests
+"on":
```

```suggestion
on:
```

(both are valid, just reads weirdly, and most yaml parsers don't have a :norway: problem anymore)